app/src/main/java/org/mozilla/fenix/compose/Text.kt

```diff
@@ -16,21 +18,28 @@ import org.mozilla.fenix.theme.FirefoxTheme
  * @param text The text to be displayed.
  * @param modifier [Modifier] to apply to this layout node.
  * @param fontSize The size of glyphs to use when painting the text.
+ * @param fontFamily The font family to be used when rendering the text. See [TextStyle.fontFamily].
```

Comment:
I am looking at the 26 files changed, and thinking let's rip some of this out into separate PRs and land smaller commits. So, we aren't always reviewing 26 files changes. 😱

The changes here are an easy candidate.